lib/validation/jsonSchemas.js

```diff
@@ -362,6 +362,7 @@ schemas[validationKeys.CONFIGURE] = {
 	'properties': {
 		'appConfigId': {'type': 'string'},
 		'concurrency': {'type': 'number'},
+		'additionalConfigFile': {'type': 'string'},
```

Comment:
I think, 'singleConfigFile': {'type': 'string'} is need to be added here?

Reply:
Actually no. `singleConfigFile` does not exist by purpose. In that case singleConfigFile would have same flaw as configFile.